docs/USAGE.md

```
@@ -0,0 +1,589 @@
+import "@lars-artmann/typespec-asyncapi";
+using AsyncAPI;
+```
+
+### Issue: Invalid AsyncAPI output
+
+**Problem:**
+Generated YAML fails AsyncAPI validation
+
+**Solutions:**
+1. Validate your TypeSpec syntax:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)